concent_api/concent_api/settings/base.py

```diff
@@ -200,6 +202,16 @@
     },
 }
 
+# Django constance config
+CONSTANCE_BACKEND = 'constance.backends.database.DatabaseBackend'
+CONSTANCE_CONFIG = {
+    # Defines if Concent is in shutdown mode.
+    'SHUTDOWN_MODE': (False, 'Shutdown mode', bool),
+}
```

Comment:
It only does initial migration, there are no changes when you change this. I think it holds this data inside database fields.

Reply:
But is the data preserved after a complex migration? It's not necessarily a problem with this single setting but it we add this plugin we're going to want to use for more and more stuff. I'd like to be sure that it can migrate settings. If not, I'd prefer to use a different one or just create the tables manually.

Reply:
The migration is inside the `django-constance` app. Concent will not create any additional migration unless you edit the original code of extra app. Which will not happen, because it is installed from requirements. I don't really see what could cause any problems here.

Reply:
We talked about it today. @rwrzesien's opinion is that the live settings are not meant to be preserved long-term so resetting them to defaults on migration is not a problem.

I'm not sure. I have no opinion one way or another yet. We'll leave it as is and start worrying about it when we have settings that we need to migrate.